QModManager/Error.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using UnityEngine.UI;
+
+namespace QModManager
+{
+    internal static class Error
+    {
+        internal static void ShowError(string error, Action onLeftButtonPressed = null, Action onRightButtonPressed = null, string leftButtonText = "See Log", string rightButtonText = "Close")
+        {
+            uGUI_SceneConfirmation confirmation = uGUI.main.confirmation;
+
+            if (onLeftButtonPressed == null) onLeftButtonPressed = () => { Process.Start(Path.Combine(QModPatcher.QModBaseDir, "../Subnautica_Data/output_log.txt")); };
+            if (onRightButtonPressed == null) onRightButtonPressed = () => { Console.Write(""); };
+
+            if (string.IsNullOrEmpty(leftButtonText)) confirmation.yes.gameObject.SetActive(false);
+            else confirmation.yes.gameObject.GetComponentInChildren<Text>().text = leftButtonText;
+            if (string.IsNullOrEmpty(rightButtonText)) confirmation.no.gameObject.SetActive(false);
+            else confirmation.no.gameObject.GetComponentInChildren<Text>().text = rightButtonText;
+
+            confirmation.Show(error, delegate (bool leftButtonClicked)
+            {
+                if (leftButtonClicked) onLeftButtonPressed.Invoke();
+                else onRightButtonPressed.Invoke();
+
+                Console.Write("");
```

Comment:
If there is a reason for this, at least say so in a comment.

Reply:
The buttons have a stupid tendency to not work 90% of the time. I made some changes that broke them but I wasn't what caused it. This line is a leftover from testing. I will remove it

Reply:
Addressed in commit 10ef4b8